.github/workflows/prober-test.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       id-token: write
+      packages: write
```

Comment:
why do we need write on packages now?

Reply:
copy/pasting leftover while debugging, it's removed now!